wrap/result.go

```diff
@@ -0,0 +1,126 @@
+package wrap
+
+import (
+	"bytes"
+	"crypto/md5"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"text/template"
+	"time"
+
+	"github.com/Songmu/wrapcommander"
+)
+
+type result struct {
+	Cmd        []string
+	Name, Memo string
+
+	Output         string `json:"-"`
+	Pid            int
```

Comment:
`Pid`, `StartAt`, `EndAt` not used? For saving in the state file?

Reply:
Drop them at e0e43228d